dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -260,6 +302,13 @@ def parse_args():
         help="Call set_index on the key column to sort the joined dataframe.",
     )
     parser.add_argument("--runs", default=3, type=int, help="Number of runs")
+    parser.add_argument(
+        "--no-ib", action="store_true", help="Disable infiniband over ucx."
+    )
+    parser.add_argument("--no-tcp", action="store_true", help="Disable tcp over ucx.")
+    parser.add_argument(
+        "--no-nvlink", action="store_true", help="Disable NVLink over ucx."
+    )
```

Comment:
I think it's usually less confusing to have variables that enable/turn on things, rather than disable/turn off. For that, I would suggest that we have variables such as `--enable-infiniband`, `--enable-tcp` and `--enable-nvlink`. This would also make things consistent with those variables in `LocalCUDACluster`.

Reply:
Of course, the default values for those would then be `True`, and we would then need two variables, `enable/disable`, as in https://github.com/rapidsai/dask-cuda/blob/branch-0.12/dask_cuda/dask_cuda_worker.py#L155-L169.

Reply:
> Of course, the default values for those would then be True, and we would then need two variables, enable/disable, as in https://github.com/rapidsai/dask-cuda/blob/branch-0.12/dask_cuda/dask_cuda_worker.py#L155-L169.

Right - I was hoping to have all transports turned on in the benchmark by default, hence the "negative" command-line options.  I am not familiar with `click` or the `enable/disable` syntax, but that might be reasonable.  Should we just change the `argparse` flags to require explicit `True/False` values (e.g. `python script.py --enable-nvlink True`)?

Reply:
I think we would be better off with something like the following. Unfortunately it allows a user to specify both (and whatever comes last is valid), but I guess that would be less cluttered for users.

```python
parser.add_argument('--enable-tcp-over-ucx', dest='enable_tcp_over_ucx', action='store_true')
parser.add_argument('--disable-tcp-over-ucx'', dest='enable_tcp_over_ucx', action='store_false')
parser.set_defaults(enable_tcp_over_ucx=True)
```

Reply:
> I think we would be better off with something like the following...

Okay - That looks fine to me. Thanks!

Reply:
Thanks for the changes Rick!